indigo_app/templates/indigo_api/_work_points_in_time.html

```diff
@@ -3,14 +3,25 @@
     <li class="timeline-item card">
       <div class="card-body">
         {% block new-amendment-card %}
-          <button class="btn btn-primary add-amendment"><i class="fas fa-plus"></i> Add amendment</button>
-
-          <form action="{% url 'new_work_amendment' frbr_uri=work.frbr_uri %}" method="POST" id="new-amendment-form">
-            {% csrf_token %}
+          <div class="btn-group">
+            <button class="btn btn-primary add-amendment"><i class="fas fa-plus"></i> Add amendment</button>
+            <form action="{% url 'new_work_amendment' frbr_uri=work.frbr_uri %}" method="POST" id="new-amendment-form">
+              {% csrf_token %}
+              <input type="hidden" name="amending_work" value="">
+              <input type="hidden" name="date" value="">
+            </form>
 
-            <input type="hidden" name="amending_work" value="">
-            <input type="hidden" name="date" value="">
-          </form>
+            <button type="button" class="btn btn-primary dropdown-toggle dropdown-toggle-split" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span class="sr-only">Toggle Dropdown</span></button>
+            <div class="dropdown-menu">
+              <form method="POST">
+                {% csrf_token %}
+                <button class="dropdown-item" type="submit" name="arbitrary_expression_date"
+                        value="{{ consolidation_date|date:'Y-m-d' }}"
+                        formaction="{% url 'new_work_arbitraryexpressiondate' frbr_uri=work.frbr_uri %}">Add consolidation date: {{ consolidation_date|date:'Y-m-d' }}</button>
```

Comment:
You can set the action directly on the form, it's not going to change between submit buttons, so it doesn't need to be `formaction` on the submit button.

Reply:
so this woud become a `submit` button?